ExportAs/ExportAs.py

```diff
@@ -77,3 +89,37 @@ def export(self, node, writerFormat):
       logging.info(f"Exported {node.GetName()} to {fileName}")
     else:
       slicer.util.errorDisplay(f"Could not save {node.GetName()} to {fileName}")
+  
+  def exportHardened(self, node, writerFormat):
+    # Clone model node
+    shNode = slicer.vtkMRMLSubjectHierarchyNode.GetSubjectHierarchyNode(slicer.mrmlScene)
+    itemIDToClone = shNode.GetItemByDataNode(node)
+    clonedItemID = slicer.modules.subjecthierarchy.logic().CloneSubjectHierarchyItem(shNode, itemIDToClone)
+    print (clonedItemID)
+    clonedNode = shNode.GetItemDataNode(clonedItemID)
+    
+    # Apply model node transform to cloned node
+    trfNode = slicer.mrmlScene.GetNodeByID(node.GetTransformNodeID())
+    clonedNode.SetAndObserveTransformNodeID(trfNode.GetID())
+
+    # Harden cloned model transform
```

Comment:
I agree with @muratmaga this should apply to all `vtkMRMLTransformableNode` instances, so we should be sure that works and reflect that in the comments instead of only mentioning models.

Reply:
As you want guys... I can check for models if you want. I did it this way because should be interesting too for other nodes like volumes after alignment... but...

Reply:
Any node class that is both Transformable and Storable should be captured by just checking the superclass (so models, volumes, fiducials, etc).